settings/ticket.rst

```diff
@@ -149,3 +149,24 @@ the desired agents in the ``Exception Users`` list.
 
       The search function in this area is only effective, if you have too many
       agents to display at once to help you with the configuration.
+
+Notifications
+-------------
+
+Default Notifications
+   This setting allows you to configure the default ticket notifications that
+   will be applied to all new agent users. Simply change the notification matrix
+   to desire state and press **Save**.
+
+   However, in case an agent has already customized their ticket notifications,
+   their choice will be preserved. Optionally, you can also **Apply current
```

Comment:
This sentence sounds as if change would be applied to agents who didn't notification configuration. Which is not the case. If configuration was not changed, it does not fallback to default.